functions.js

```diff
@@ -148,3 +148,20 @@ document.getElementById('featured-button').addEventListener('click', () => {
 document.getElementById('popup-close').addEventListener('click', () => {
   openClosePopup();
 });
+function validateEmail(email, error, event) {
+  if (email !== email.toLowerCase()) {
+    event.preventDefault();
+    const msg = document.getElementById('error-message');
+    msg.innerText = error;
+    msg.style.color = 'red';
+    msg.style.fontSize = '12px';
+  }
+}
+const form = document.getElementById('contact-me-form');
+form.addEventListener('submit', (event) => {
+  // handle the form data
```

Comment:
- [ OPTIONAL] Please do not leave unused code as a comment in your codebase. unused code as comments makes your code not clean.

Reply:
Thanks!